zubhub_frontend/zubhub/src/views/create_team/CreateTseam.jsx

```diff
@@ -0,0 +1,453 @@
+  const [completedSteps, setcompletedSteps] = useState([]);
+  const { height } = useDomElementHeight('navbar-root');
+  const isSmallScreen = useMediaQuery(theme => theme.breakpoints.down('sm'));
+  const wizardRef = useRef(null);
+  const classes = makeStyles(createProjectStyle)({ height });
+  const commonClasses = makeStyles(styles)();
+  const [draftStatus, setDraftStatus] = useState(DRAFT_STATUSES.idle);
+  const [activeStep, setActiveStep] = useState(1);
+  const [state, setState] = useState({ ...JSON.parse(JSON.stringify(script.vars.default_state)) });
+  const [publishOrAddTags, setPublishOrAddTags] = useState(false);
+  const [addTagsDialog, setAddTagsDialog] = useState(false);
+  const [value, setValue] = useState('');
+  const [remoteTags, setRemoteTags] = useState([]);
+  const [popularTags, setPopularTags] = useState(script.testTags);
+  const [mode, setMode] = useState('');
+  const [preview, setPreview] = useState(false);
+
+  const isActive = index => index + 1 === activeStep;
+  const isCompleted = index => completedSteps.includes(index + 1);
+  const togglePublishOrAddTags = () => {
+    formik.values?.tags.length == 0 ? setPublishOrAddTags(!publishOrAddTags) : toggleAddTagsDialog();
+  };
+  const toggleAddTagsDialog = () => setAddTagsDialog(!addTagsDialog);
+  const clearSuggestions = () => setRemoteTags([]);
+
+  const handleSetState = obj => {
+    if (obj) {
+      Promise.resolve(obj).then(obj => {
+        setState(state => ({ ...state, ...obj }));
+      });
+    }
+  };
+
+  const getToastMessage = () => {
+    let message = '';
+    if (activeStep === 1 && props.match.path === '/projects/create') {
+      message = 'createProject.addedToDraft';
+    }
+    if ([1, 2].includes(activeStep) && props.match.params.id) {
+      message = 'createProject.savedStep';
+    }
+    if (activeStep === 3 && props.match.params.id) {
+      message = 'createProject.createToastSuccess';
+    }
+    return message;
+  };
+
+  const handleChangeTag = async value => {
+    setValue(value);
+    script.searchTags(value, (error, data) => {
+      if (!error) setRemoteTags(data);
+    });
+  };
+
+  const addTag = value => {
+    const values = [...formik.values.tags, value];
+    formik.setFieldValue('tags', values);
+    clearSuggestions();
+    setValue('');
+  };
+
+  const removeTag = tagIndex => {
+    const tags = [...formik.values.tags].filter((_, index) => index !== tagIndex);
+    formik.setFieldValue('tags', tags);
+  };
+
+  useEffect(() => {
+    if (props.match.params.id) {
+      // Promise.all([script.getProject({ ...props, ...formik }, state), script.getCategories(props)]).then(result =>
+      //   handleSetState({ ...result[0], ...result[1] }),
+      // );
+    } else {
+      // handleSetState(script.getCategories(props));
+    }
+    const params = new URLSearchParams(window.location.search);
+    const queryParams = Object.fromEntries(params.entries());
+    if ('mode' in queryParams) setMode(queryParams.mode);
```

Comment:
entries is not required use map instead.

```suggestion
    if (params.has('mode')) setMode(params.get('mode'));
```